obspy/imaging/spectrogram.py

```diff
@@ -104,14 +104,19 @@ def spectrogram(data, samp_rate, per_lap=0.9, wlen=None, log=False,
 
     # set wlen from samp_rate if not specified otherwise
     if not wlen:
-        wlen = samp_rate / 100.
+        wlen = 128 / samp_rate
```

Comment:
Yes, this was really crazy beforehand. The doc string needs to be adapted. (default: 128 samples)